one_fm/custom/assignment_rule/shift_request_draft.json

```diff
@@ -6,8 +6,8 @@
   "description": "Automatic Assignment",
   "is_assignment_rule_with_workflow": 0,
   "assign_condition": "workflow_state ==\"Draft\" and creation < modified",
-  "unassign_condition": "workflow_state in ('Pending Approver', )",
-  "close_condition": "workflow_state in ('Pending Approver', )",
```

Comment:
Review the ba site and the process drawing in lucid and make sure the workflow_state value